### PR TITLE
Web3-less Ether type

### PR DESCRIPTION
**Pull request by D4nte**
Step 1 of #1848

### CODE REVIEW

File: cnd/src/db/save.rs [hunk 1]
```diff
@@ -97,7 +97,7 @@ impl Save<Request<Bitcoin, Ethereum, asset::Bitcoin, asset::Ether>> for Sqlite {
             bitcoin_network: Text(alpha_ledger.network),
             ethereum_chain_id: U32(beta_ledger.chain_id.into()),
             bitcoin_amount: Text(Satoshis(alpha_asset.as_sat())),
-            ether_amount: Text(DecimalU256(beta_asset.wei())),
+            ether_amount: Text(DecimalU256(beta_asset.to_u256())),
```
Comment (D4nte) [inline cnd/src/db/save.rs]: The usage of `U256` as a middle man will be removed in a follow-up PR.

File: cnd/src/asset/mod.rs [hunk 2]
```diff
@@ -36,7 +25,7 @@ impl Asset for Ether {}
 
 impl Asset for Erc20 {}
 
-#[derive(Clone, Copy, Derivative, PartialEq)]
```
Comment (thomaseizinger) [inline cnd/src/asset/mod.rs]: Sad that all of these `Copy` implementations go away :(

Comment (D4nte) [reply cnd/src/asset/mod.rs]: I know!

File: cnd/src/asset/ether.rs [hunk 3]
```diff
@@ -1,59 +1,126 @@
...
+    fn from_wei(wei: BigUint) -> Self {
+        Ether(wei)
+    }
+}
+
+impl TryFromWei<&str> for Ether {
+    type Err = ParseBigIntError;
+
+    fn try_from_wei(string: &str) -> Result<Ether, Self::Err> {
+        let uint = BigUint::from_str(string)?;
```
Comment (D4nte) [inline cnd/src/asset/ether.rs]: Need to check we are not over U256!

**Comment by thomaseizinger** [comment]
bors retry

**Pull request merged**